crates/ide-completion/src/completions/flyimport.rs

```diff
@@ -395,6 +395,14 @@ fn import_assets_for_path(
     potential_import_name: &str,
     qualifier: Option<ast::Path>,
 ) -> Option<ImportAssets> {
+    let _p = tracing::span!(
+        tracing::Level::INFO,
```

Comment:
These could just be `tracing::info_span!` instead, though that adds additional nested macro calls so maybe not the best idea